template/src/components/icons/SpinnerIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+type Props = React.SVGProps<SVGSVGElement>;
+export function SpinnerIcon(props: Props) {
```

Comment:
Side note, is this something Radix Icons can solve?

I would like avoid add custom icons when Radix that is already imported can solve.

cc @Sneh1999 @Yuripetusko

Reply:
Nope, I looked for the appropriate icon there and it's not there. [lucide-react](https://www.npmjs.com/package/lucide-react) is very popular now for icons, and I use it on some projects, so is react-icons (although this one sometimes has problems with tree-shaking, but maybe they already solved it)

Reply:
Great context